www/app/app.ts

```diff
@@ -150,10 +150,13 @@ export class App {
     } else if (e instanceof errors.OperationTimedOut) {
       messageKey = 'error-timeout';
     } else if (e instanceof errors.ShadowsocksStartFailure) {
-      // TODO: only display this message if running on Windows.
-      messageKey = 'outline-plugin-error-antivirus';
-      buttonKey = 'fix-this';
-      buttonLink = 'https://s3.amazonaws.com/outline-vpn/index.html#/en/support/antivirusBlock';
+      if (this.isWindows()) {
+        messageKey = 'outline-plugin-error-antivirus';
+        buttonKey = 'fix-this';
+        buttonLink = 'https://s3.amazonaws.com/outline-vpn/index.html#/en/support/antivirusBlock';
+      } else {
+        messageKey = 'outline-plugin-error-generic-start-failure';
```

Comment:
Looking at messages JSON, there doesn't seem to be a substantial difference between these three:
```json
  "outline-plugin-error-generic-start-failure": "Something happened and we couldn\u2019t connect. If this happens again, please submit feedback.",
  "outline-plugin-error-unexpected": "Sorry, an unexpected error occurred. If this happens again, please submit feedback.",
  "error-unexpected": "Sorry, an unexpected error occurred. Quitting and restarting may help. If this happens again, please submit feedback."
```
Currently, this function "falls through" to `error-unexpected`; what would you think of nuking `outline-plugin-error-generic-start-failure` (currently unused, I think) and using `error-unexpected` for this instead? (line 152 would become `else if (e instanceof errors.ShadowsocksStartFailure && this.isWindows()) {`

(actually I think we could kill `outline-plugin-error-unexpected` in the same way - WDYT?)

Reply:
Sounds good! Done in de2bfcc and acb7193. PTAL.